python/cudf/cudf/core/buffer/weakrefable_buffer.py

```diff
@@ -0,0 +1,324 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Given that the main export from this module is a type called `RefCountableBuffer`, can we rename it to `ref_countable_buffer.py`?